bootstrap.php

```diff
@@ -179,20 +179,16 @@ function ($cdbid) use ($app) {
 $container->addServiceProvider(new OfferSecurityServiceProvider());
 $container->addServiceProvider(new OrganizerSecurityServiceProvider());
 
-$app['cache'] = $app->share(
-    function (Application $app) {
-        return function ($cacheType) use ($app) {
-            return new Doctrine\Common\Cache\PredisCache(new Predis\Client(
-                $app['config']['cache']['redis'],
-                [
-                    'prefix' => $cacheType . '_',
-                ]
-            ));
-        };
-    }
+$container->addShared(
+    'cache',
+    fn ($cacheType) => new Doctrine\Common\Cache\PredisCache(
```

Comment:
You will need to wrap this in `new League\Container\Argument\Literal\CallableArgument()`, otherwise the container will try to call this function to instantiate the `cache` service instead of just returning the callable as-is. See https://container.thephpleague.com/4.x/argument-types/

Reply:
Good catch, I hope this is the first one I encountered.